enzyme/Enzyme/MLIR/Passes/EnzymeMLIRPass.cpp

```diff
@@ -36,7 +36,7 @@ struct DifferentiatePass : public DifferentiatePassBase<DifferentiatePass> {
 
   template <typename T>
   void HandleAutoDiff(SymbolTableCollection &symbolTable, T CI) {
-    std::vector<DIFFE_TYPE> constants;
```

Comment:
From convo with @pengmai , @umatin could we restore this to using the same enums as LLVMEnzyme?